doc/CodeDocumentation.conf.in

```diff
@@ -698,7 +698,7 @@ CITE_BIB_FILES         =
 # messages are off.
```

Comment:
Just an idea: can we use our MFEM logo as the `PROJECT_LOGO` on line 54 of this file? It might make the produced documentation look nicer.

Reply:
Hmm, would we need to put the the png into the repo for this to work?

Reply:
Probably. It's 33k, so it's not huge.